docs/architecture/modernize-desktop/whats-new-dotnet-core-desktop.md

```diff
@@ -0,0 +1,149 @@
+
+For your brand-new desktop applications, we recommend starting directly on .NET Core. It is lightweight, runs side-by-side, high performance and cross-platform and fits perfectly on containers and microservices architectures.
+
+![.NET Framework and .NET Core](./media/whats-new-dotnet-core-desktop/dotnet-framework-and-dotnet-core.png)
+
+## .NET Standard vs. PCL
+
+The [.NET Standard](https://github.com/dotnet/standard) is a formal specification of .NET APIs that are intended to be available on all .NET implementations. The motivation behind the .NET Standard is establishing greater uniformity in the .NET ecosystem. .NET Standard is a specification of .NET APIs that make up a uniform set of contracts to compile your code against. These contracts are implemented in each .NET flavor, thus enabling portability across different .NET implementations.
+
+The .NET Standard enables the following key scenarios:
+
+- Defines uniform set of base class libraries APIs for all .NET implementations to implement, independent of the workload.
+- Enables developers to produce portable libraries that are usable across .NET implementations, using this same set of APIs.
+- Reduces or even eliminates conditional compilation of shared source due to .NET APIs, only for OS APIs.
+
+.NET Standard is the evolution of PCLs and here are the fundamental differences between .NET Standard and PCLs:
+
+- .NET Standard is a set of curated APIs, picked by Microsoft, PCLs are not.
+- The APIs that a PCL contains are dependent on the platforms that you choose to target when you create a PCL. This makes a PCL only sharable for the specific targets that you choose.
+- .NET Standard is platform-agnostic, it can run anywhere, on Windows, macOS, Linux, and so on.
+- PCLs can also run cross-platform, but they have a more limited reach. PCLs can only target a limited set of platforms.
+
+## .NET Core 3.0 new Desktop features
+
+### Support for Windows Forms and WPF
+
+Windows Forms and WPF are part of .NET Core 3.0. Both presentation frameworks are only for the Windows OS and they are not cross-platform. You can think of WPF as a rich layer over DirectX and Windows Forms as a thinner layer over GDI+. WPF and Windows Forms do a great job of exposing and exercising much of the desktop application functionality in Windows. With .NET Core 3.0. Windows Forms and WPF become available for .NET Core as well as for .NET Framework. Now you can start your new desktop applications targeting .NET Core and migrate your existing ones from .NET Framework to .NET Core.
+
+A new version of .NET Standard, version 2.1, was released at the same time. As expected, all new .NET Standard APIs are part of .NET Core 3.0.
+
+You can build desktop applications with C#, F#, and VB with .NET Core 3.0.
```

Comment:
did you ship the templates for all of these? If not, it would be good to call out, no?

Reply:
Removed the line because right now only C# makes sense, and we talked about desktop apps available on Core already